src/rtk/features/overviewOrg/overviewOrgSlice.ts

```diff
@@ -148,14 +148,13 @@ const createCopyOrg = (org: OverviewOrg) => {
   };
 };
 
-export const fetchOverviewOrgsOutbound = createAsyncThunk(
-  'overviewOrg/fetchOverviewOrgsOutbound',
+export const fetchOverviewOrgsOffered = createAsyncThunk(
```

Comment:
Either use same vocabulary as introduced in apiDelegation/given or change the folder name from apiDelegation/given => apiDelegation/offered

```suggestion
export const fetchOverviewOrgsGiven = createAsyncThunk(
```

Reply:
Nevermin I can fix this in my branch. Going to nemae to offered.